packages/react-devtools-shared/src/utils.js

```diff
@@ -45,6 +45,7 @@ import {
 import {localStorageGetItem, localStorageSetItem} from './storage';
 import {meta} from './hydration';
 import type {ComponentFilter, ElementType} from './types';
+import {symbol} from 'prop-types';
```

Comment:
?

Reply:
@gaearon Shame :/ I guess my editor auto complete added it when writing [symbol.iterator], I will review my editor config.

Reply:
![Screenshot 2020-09-22 at 11 35 46](https://user-images.githubusercontent.com/51530311/93860437-c9a5d000-fcc7-11ea-94ce-d4e410f6b97a.png)